src/test/kotlin/assimp/obj/obj.kt

```diff
@@ -11,8 +11,41 @@ class obj : StringSpec() {
         "cube"{ cube(path + "cube.obj") }
         "wall"{ wall(path + "wall.obj") }
         "box"{ box(path + "box.obj") }
-        "spider"{ spider(path + "spider.obj") }
-        "nanosuit" { nanosuit(path + "nanosuit/nanosuit.obj") }
-        "shelter" { shelter(path + "statie B01.obj")}
+
+        "spider"{ spider(path + "spider.obj",
```

Comment:
not sure about this. the iosystem should be able to resolve these and in my opinion it should be up to the iosystem to load it for assimp when it is requested. I could be reading this wring though, so please explain

Reply:
The default IO system is able to resolve those paths. The IOSystem for readFromMemory however can only resolve paths that where added with a buffer to the IOSystem.
I add those paths for the tests so that I can test that readFromMemory is working. That's why I specify all files here and not just the base path

Reply:
The other option would be to create an IOSystem for the tests that can resolve paths when requested even while simulating readFromMemory, but I don't think this will help. It would just add a third IOSystem which needs to be maintained and tested